_posts/2016-05-16-rust-at-one-year.md

```diff
@@ -0,0 +1,268 @@
+  [Cargo], gained numerous abilities this year, most notably the
+  [install subcommand].
+
+[lldb]: http://lldb.llvm.org/
+[gdb]: https://www.gnu.org/software/gdb/
+[perf]: https://perf.wiki.kernel.org/index.php/Main_Page
+[valgrind]: http://valgrind.org/
+[callgrind]: https://kcachegrind.github.io/html/Home.html
+[enrich the experience]: https://michaelwoerister.github.io/2015/03/27/rust-xxdb.html
+[gdb]: https://sourceware.org/ml/gdb-patches/2016-04/msg00570.html
+[cargo profile]: http://www.suchin.co/2016/05/11/Introducing-Cargo-Profiler/
+[IDEs]: https://www.rust-lang.org/ides.html
+[Racer]: https://github.com/phildawes/racer
+[numerous IDE plugins]: https://areweideyet.com/
+[rustfmt]: https://github.com/rust-lang-nursery/rustfmt
+[official style]: https://github.com/rust-lang/rfcs/pull/1607
+[Cargo]: http://blog.rust-lang.org/2016/05/05/cargo-pillars.html
+[install subcommand]: http://blog.rust-lang.org/2015/12/10/Rust-1.5.html
+
+- **Compiler**. We've seen some [across-the-board improvements] to compile
+  times, and now offer [parallelized code generation][parallel codegen] for
+  further speedups. But the biggest wins will come from the ongoing work on
+  [incremental compilation], which will minimize the amount of work the needed
+  when recompiling code after editing it. A vital step here was the move to a
+  [custom intermediate representation][MIR], which has many other benefits as
+  well. Another focus has been errors, including
+  [detailed explanations of most errors][error index], and ongoing work to
+  [improve the clarity and focus of errors][error format]. Expect to hear more
+  on both fronts soon.
+
+[across-the-board improvements]: http://blog.rust-lang.org/2015/06/25/Rust-1.1.html
+[parallel codegen]: http://blog.rust-lang.org/2015/08/06/Rust-1.2.html
+[incremental compilation]: https://github.com/rust-lang/rfcs/pull/1298
+[MIR]: http://blog.rust-lang.org/2016/04/19/MIR.html
+[error index]: https://doc.rust-lang.org/error-index.html
+[error format]: https://internals.rust-lang.org/t/new-error-format/3438
+
+- **Core language**. We've kept one list purposefully short this year: growth in
+  the core language. While we have some important features in the pipeline (like
+  [more flexible borrowing rules] and [specialization]), [Rust users] by and
+  large are happy with the core language and prefer the community to focus on
+  the ecosystem and tooling.
+
+[Rust users]: https://internals.rust-lang.org/t/production-user-research-summary/2530
+[more flexible borrowing rules]: http://smallcultfollowing.com/babysteps/blog/2016/04/27/non-lexical-lifetimes-introduction/
+[specialization]: https://github.com/rust-lang/rfcs/pull/1210
+
+There's a lot more to say about what's happened and what's coming up in the Rust
+world&mdash;over the coming months, we'll be using this blog to say it.
+
+### Rust in community
+
+It turns out that people like to get together and talk Rust:
+
+- August 2015: a sold-out [RustCamp];
```

Comment:
I sort of feel that the upcoming events should be listed separately, perhaps mention the RustCamp in the sentence itself and follow it with "There are numerous rust-specific conferences coming up, \<list>"